cmake/options.cmake

```diff
@@ -69,6 +69,16 @@ option(OSQUERY_NO_DEBUG_SYMBOLS "Whether to build without debug symbols or not,
 
 option(BUILD_TESTING "Whether to enable and build tests or not")
 
+# Linux can use source and formula modules to link dependencies; this
+# feature is not yet available on Windows and macOS
+if(DEFINED PLATFORM_LINUX)
```

Comment:
I was thinking of only using the new dependencies on Linux for the first release, since it's where we are currently blocked. We can enable the new modules on macOS and Windows too in the second release